### PR TITLE
Pom 133 manual parole review date

### PR DESCRIPTION
**Pull request by starswan**
This ticket just adds parole_review_date to case_information. No behavioural changes

### CODE REVIEW

File: app/controllers/case_information_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class CaseInformationController < PrisonsApplicationController
+  before_action :set_case_info, only: [:edit, :edit_prd, :show]
```
Comment (rossjones) [inline app/controllers/case_information_controller.rb]: and :update?

File: app/models/case_information.rb [hunk 2]
```diff
@@ -27,4 +27,8 @@ class CaseInformation < ApplicationRecord
   # nil means MAPPA level is completely unknown.
   # 0 means MAPPA level is known to be not relevant for offender
   validates :mappa_level, inclusion: { in: [0, 1, 2, 3], allow_nil: true }
+
+  acts_as_gov_uk_date :parole_review_date
+
+  validates :parole_review_date, date: { after: proc { Date.yesterday }, allow_nil: true }
```
Comment (rossjones) [inline app/models/case_information.rb]: What is Date.yesterday for?

File: db/schema.rb [hunk 3]
```diff
@@ -134,6 +135,15 @@
     t.index ["nomis_staff_id"], name: "index_pom_details_on_nomis_staff_id"
   end
 
+  create_table "responsibilities", force: :cascade do |t|
```
Comment (rossjones) [inline db/schema.rb]: Accidentally included?

Comment (starswan) [reply db/schema.rb]: yes good spot

File: app/models/case_information.rb (hunk 2 above)
Comment (starswan) [reply app/models/case_information.rb]: to make sure that we don't enter a date in the past

Comment (rossjones) [reply app/models/case_information.rb]: TIL

**Pull request merged**